keys/marcello.thiry.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAoTf/hH5+G5fqKUtP8rcVMdysRDnijH47HZx47/qCTu marcello.thiry@gmail.com
```

Comment:
The email should end with @goodeggs.com to maintain the scope of goodeggs accounts. Can you update it?